velox/runner/LocalRunner.cpp

```diff
@@ -120,9 +120,6 @@ void LocalRunner::waitForCompletion(int32_t maxWaitUs) {
   auto startTime = getCurrentTimeMicro();
   for (auto& future : futures) {
     auto& executor = folly::QueuedImmediateExecutor::instance();
-    if (getCurrentTimeMicro() - startTime > maxWaitUs) {
```

Comment:
We should keep this as we don't want to wait for more than maxWaitUs in total. We shall just bump up the limit in test